docs/demo-generic-agent.md

```diff
@@ -0,0 +1,205 @@
+
+---
+
+## Import New Agent
+
+To deploy the Generic Agent:
+
+1. Navigate to [Import New Agent](http://kagenti-ui.localtest.me:8080/Import_New_Agent#import-new-agent) in the Kagenti UI.
+2. In the **Select Namespace to Deploy Agent** drop-down, choose the `<namespace>` where you'd like to deploy the agent. (These namespaces are defined in your `.env` file.)
+3. Under [**Select Environment Variable Sets**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#select-environment-variable-sets):
+   - `ollama` or `openai`
+4. Under [**Environment Variable**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#select-environment-variable-sets), select:
+   - Click `Add Environment Variable`
+   - Under `Name` put `MCP_URLS` and under `Value` put `http://movie-tool:8000/mcp, http://flight-tool:8000/mcp`
+5. In the **Agent Source Repository URL** field, use the default:
+   <https://github.com/kagenti/agent-examples>
+   Or use a custom repository accessible using the GitHub ID specified in your `.env` file.
+6. For **Git Branch or Tag**, use the default `main` branch (or select another as needed).
+7. Set **Protocol** to `a2a`.
+8. Under [**Specify Source Subfolder**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#specify-source-subfolder):
+   - Click `Select from examples`
+   - Choose: `a2a/geneic_agent`
+9. Click **Build & Deploy New Agent** to deploy.
+
+**Note:** The `ollama` environmental variable set specifies `llama3.2:3b-instruct-fp16` as the default model. To download the model, run `ollama pull llama3.2:3b-instruct-fp16`. Please ensure an Ollama server is running in a separate terminal via `ollama serve`. 
+
+---
+
+## Import New Tool
+
+To deploy the Movie Tool:
+
+1. Go to [OMDB's website](https://www.omdbapi.com/) and apply for a free API key
+1. Navigate to [Import New Tool](http://kagenti-ui.localtest.me:8080/Import_New_Tool#import-new-tool) in Kagenti's UI.
+1. Select the same `<namespace>` as used for the agent.
+1. Under [**Environment Variable**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#select-environment-variable-sets), select:
+   - Click `Add Environment Variable`
+   - Under `Name` put `OMDB_API_KEY` and under `Value` put your OMDB API key
+1. Use the same source repository:
+   <https://github.com/kagenti/agent-examples>
+1. Choose the `main` branch or your preferred branch.
+1. Set **Select Protocol** to `streamable_http`.
+1. Under **Specify Source Subfolder**:
+   - Select: `mcp/movie_tool`
+1. Click **Build & Deploy New Tool** to deploy.
+
+To deploy the Flight Tool:
+
+1. Navigate to [Import New Tool](http://kagenti-ui.localtest.me:8080/Import_New_Tool#import-new-tool) in the UI.
+1. Select the same `<namespace>` as used for the agent.
+1. Use the same source repository:
+   <https://github.com/kagenti/agent-examples>
+1. Choose the `main` branch or your preferred branch.
+1. Set **Select Protocol** to `streamable_http`.
+1. Under **Specify Source Subfolder**:
+   - Select: `mcp/flight_tool`
```

Comment:
Same with the agent as above, we need to [edit the UI](https://github.com/kagenti/kagenti/blob/0a2d4b653daa22ef8ec481c9ef0d114672160d95/kagenti/ui/pages/05_Import_New_Tool.py#L25) or edit the instructions here to "manually input" or whatever the equivalent was